packages/babel-generator/src/printer.ts

```diff
@@ -586,8 +586,6 @@ class Printer {
   ) {
     if (!node) return;
 
-    this._endsWithInnerRaw = false;
```

Comment:
Will there be a case where `true` is here? (when this node follows a token)
I'm worried about accidentally printing before `(`.

Reply:
Good call, `1 + (( /* c */ ) => {})`